src/Database/Schema/SqlserverSchema.php

```diff
@@ -357,6 +361,7 @@ public function columnSql(TableSchema $schema, string $name): string
             TableSchema::TYPE_BIGINTEGER => ' BIGINT',
             TableSchema::TYPE_BINARY_UUID => ' UNIQUEIDENTIFIER',
             TableSchema::TYPE_BOOLEAN => ' BIT',
+            TableSchema::TYPE_CHAR => ' CHAR',
```

Comment:
This should be `NCHAR` not `CHAR`. The test shouldn't have been updated to use `nchar` as that isn't an abstract type the ORM supports.

Reply:
Cool!
I have updated it.
Maybe, would be wise to put a note in docs... that orm maps to unicode d types and not non unicode like `char` as considered by sql server or is that an implied understanding with users who use sql server?